### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a:nth-child(2))

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1171]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
.mxgrid__item__bg__p:nth-child(2) > a:nth-child(2)
#### Snippet
`<a href="https://gitlab.matrix.org/matrix-org/olm/blob/master/docs/">open standard</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.96 (foreground color: #0098d4, background color: #f4f4f4, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
